spec/RxRelay.spec.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Relay } from '../dist/RxRelay';
+
+describe('RxRelay', () => {
+  it('should not import all of RxJS', () => {
+    expect(Relay.prototype.zip).toBe(undefined);
+    expect(Relay.prototype.mergeMap).toBe(undefined);
+  });
+});
```

Comment:
This is used to roughly confirm it doesn't import all of rxjs and doesn't later regress. Works because if all of rxjs was imported, Relay would inherit these operators from Observable